src/components/header/SubHeader.js

```diff
@@ -326,6 +327,15 @@ class SubHeader extends Component {
         caption: counterpart.translate('window.Delete.caption'),
         hotkey: keymap.DELETE_DOCUMENT,
       },
+      {
+        action: 'comments',
+        handler: () => {
```

Comment:
Can't we just use `handler: handleComments` instead of creating an anonymous function ? I just noticed we're doing the same for other actions but I don't really see a reason (unless I'm missing something obvious).

Reply:
Yeah you are right. Changed it. Thanks